src/nrc/tests/admin/test_notifications.py

```diff
@@ -145,3 +169,96 @@ def test_resend_notification_action(self, mock_deliver_message):
         # Remaining response should belong to the Notification that was not selected
         notificatie3.refresh_from_db()
         self.assertEqual(notificatie3.notificatieresponse_set.count(), 1)
+
+    def test_create_notification_as_cloudevent(self, mock_deliver_cloudevent, _):
+        response = self.app.get(
+            reverse("admin:datamodel_notificatie_add"), user=self.user
+        )
+
+        form = response.forms["notificatie_form"]
+
+        form["forwarded_msg"] = json.dumps(self.forwarded_msg, default=str)
+        form["kanaal"] = self.kanaal.pk
+
+        response = form.submit()
+
+        self.assertEqual(response.status_code, 302)
+
+        # Cloudevent should be sent
+        mock_deliver_cloudevent.assert_called_once_with(
+            self.cloudevent_abonnement.id,
+            self.expected_cloudevent
+            | {"id": mock_deliver_cloudevent.call_args[0][1]["id"]},
+            notificatie_id=Notificatie.objects.get().id,
+            attempt=1,
+        )
+
+        # Verify that only one Notificatie was created (via the admin)
+        self.assertEqual(Notificatie.objects.count(), 1)
```

Comment:
Does sending a cloud event like this (by using the `send_cloudevents` toggle) result in a NotificatieResponse being created, or a CloudEventResponse?